code-of-conduct-transparency-reports/2024-07-01-coc-transparency-report.md

```diff
@@ -0,0 +1,33 @@
+# The Carpentries Code of Conduct Transparency Report
+
+2024-07-01
+
+## Overview
+
+### Reports
+
+Between 2024-04-01 and 2024-06-30, the Code of Conduct committee processed NO formal incident report. 
+
+### Potential Code of Conduct Breaches
+
+There was NO report of an incident.
+
+### Summary of Police Matters
+
+There were NO police matters.
+
+### Policy changes
+
+There were NO policy changes.  
+ 
+### Committee changes
+
```

Comment:
Need to add that we have a new liaison to the executive council.

Reply:
Please add that Jannetta Steyn joined the committee as a new liaison to the Board of Directors.